K2Bridge/Factories/BucketFactory.cs

```diff
@@ -19,7 +25,7 @@ internal static class BucketFactory
         /// </summary>
         /// <param name="row">The row to be transformed to bucket.</param>
         /// <returns>A new DateHistogramBucket.</returns>
-        public static DateHistogramBucket CreateDateHistogramBucketFromDataRow(DataRow row)
+        public static DateHistogramBucket CreateDateHistogramBucketFromDataRow(DataRow row, ILogger logger)
```

Comment:
I think we need to favor consistency. we can have the logger here but it needs to match the rest of the code. other buckes for example are not getting the logger arg

Reply:
done